asset/asset.go

```diff
@@ -569,6 +606,10 @@ func (id PrevID) Hash() [sha256.Size]byte {
 	return *(*[sha256.Size]byte)(h.Sum(nil))
 }
 
+// OutPoint is a type alias for an asset outpoint. It links the anchor outpoint
+// to the asset it secures.
+type OutPoint = PrevID
```

Comment:
`PrevOut`?

Reply:
Or `AnchorPoint` (to be used as `asset.AnchorPoint` which reads nicely IMO) to not confuse it with a purely on-chain `wire.OutPoint`?